rules-topics/21_status-effects.md

```diff
@@ -107,33 +110,76 @@ Skilless refers to any abilities purchased with Skill Points. You do not lose an
 | UNDEAD| Target is visibly a lesser undead, may talk, move, and use skills for x time. | Varies by source. |Yes |
-| WEAKEN| Target damage is lowered by x amount for y time. | Varies by source. | No |
+| WEAKEN| Target damage is lowered by x amount for y time. | Varies by source. | Yes |
  
 ## Effects
 
 ---
 | Term| Description |Encounter Call |Visible?|
 |:---|-----|:----:|:---:|
-| DEATH | Target immediately takes DEAD status. | Varies by source. | No |
+| DEATH | If the source of this effect would reduce the target to 0 or less Body, target immediately takes DEAD status. | Varies by source. | No |
 | DECIMATE | Target takes DEAD status upon receiving this effect. If target dissipates and resurrects, target incurs double Strain. (4 Total) | (Type) Decimate | No |
 | DODGE | Ability user may resist more than one type of effect, and so dodges it. | Dodge | No |
-| DOOM | Target is reduced to -1 body. | Varies by source. | No |
+| DOOM | Target takes X Damage, some creatures may be weak or resistant to this. | Varies by source. | No |
 | LIFE | Target is healed to full body and all status effects of target's choice are removed. This works on DEAD targets. | Varies by source. |Yes |
 | MUMMY CURSE | Target immediately dissipates and seeks resurrection. | Killing Blow 1, 2, 3, Mummy Curse | No |
 | REFLECT | Ability user may reflect x ability. The reflected ability may be resisted, but may not be reflected again. | Varies by source. | Not Applicable |
 | RESIST| Ability user resists x effect. |Resist | Not Applicable |
 | SOLIDIFY | Ability user or Target's GASEOUS effect is removed. | Varies by source. | Not Applicable |
 | SPELLSTRIKE | Ability user may deliver a spell with a physical weapon. |Spellstrike (Spell Name) | Not Applicable |
 | VOICE RADIUS| All targets that hear the call take X effect. | Voice Radius X| No |
+
+## How to Diagnose
+
+ 
+|Term|How to Diagnose|
+|:---|:---:|
+|ANIMATE|Identify|
+|BERSERK|Visible|
+|BLEED OUT|First Aid|
+|BLIND|First Aid|
+|CONCENTRATION|Visible|
+|CONFINE|Visible|
+|CONFUSE|Visible|
+|CONSCIOUS|First Aid|
+|CURSE|Identify|
+|CURSE OF TRANSFORMATION|**Identify|
+|DEAD|First Aid|
+|DISARM|Visible|
+|DRAINED|First Aid|
+|ENGULF|Visible|
+|ENSLAVEMENT|Break Enslavement Rune|
+|ENTANGLE|Visible|
+|FEAR|Visible|
+|INERT|First Aid|
+|INVULNERABLE|Visible|
+|KNEEL|Visible|
+|PRESERVE|First Aid|
+|RIFT LOCK|Visible|
+|SILENCE|First Aid|
+|SLEEP|First Aid|
+|SLOW|Visible|
+|STASIS|Visible|
+|STOP THRUST|Visible|
+|STUN|Visible|
+|UNCONSCIOUS|First Aid|
+|UNDEAD|Visible|
+|WEAKEN|Visible|
+|INFECT|Identify|
+|REGENERATE|Visible|
+|REVIVE|First Aid|
+|SPIRIT BOTTLE|Identify|
+|TRANSFORMS|Can not be Identified|
+ 
+**Some Curse of Transformations and transforms will have a physical description that a player can see but cannot confirm be confirmed as a visible effect.
```

Comment:
This is... not true. If you can see it, it is visible. 

So if someone has pale skin, they have visibly pale skin. A Dark Elf is "a pale-skinned Elf"

Reply:
I wanted to avoid confusion.  If we say that a CoT is only able to be detected by identified it makes it sound like there are no visible effects.  I am happy to reword but trying to make it clear.

Reply:
I see what the concern is I wrote it like a drunk person.  Updated to **Some Curse of Transformations and transforms will have a physical description that a player can see but cannot confirm as a part of a CoT or Transformation.